kubetest/anywhere.go

```diff
@@ -241,7 +241,11 @@ func (k *kubernetesAnywhere) writeConfig(configTemplate string) error {
 }
 
 func (k *kubernetesAnywhere) Up() error {
-	cmd := exec.Command("make", "-C", k.path, "WAIT_FOR_KUBECONFIG=y", "deploy")
+	cmd := exec.Command("make", "-C", k.path, "setup")
```

Comment:
@krzyzacy @neolit123 we just need to merge them around the same time and follow with an immediate bump, since this runs an extra setup step /shrug

I can co-ordinate on this.

Reply:
@BenTheElder please review my makefile + bash :fire: in https://github.com/kubernetes/kubernetes-anywhere/pull/581 and you can LGTM.

kubeadm-gce-master runs at 2h rate, but we can have a red run or two if needed.

also i'm anticipating red runs anyhow, unless i magically got it right the first time. :smile: